docs/topics/mta-review-reports.adoc

```diff
@@ -152,6 +155,62 @@ image::3-5-expanded-issue.png[Expanded rule in the issues report]
 
 Issues are sorted into four categories by default. Information on these categories is available at ask Category.
 
+
+=== Insights
+
+:FeatureName: Insights
+include::snippets/technology-preview-admonition.adoc[]
+
+Previously a violation generated by a rule with zero effort was listed as an issue in the static report. This is now listed as an insight instead. Issues are generated by general rules, whereas string tags are generated by tagging rules. String tags indicate the presence of a technology but do not show the code location. With the introduction of Insights, you can see the technology used in the application along with its usage in the code.
+
+For example, a rule searching for deprecated API usage in the code that does not impact the current migration but can be tracked and fixed when needed in the future.
+
+Unlike issues, insights do not need to be fixed for a successful migration. They are generated by any tagging or non-tagging rule and do not have an effort and category assigned. They might have a message and tag.

```

Comment:
can we make this simpler and say "generated by any rule that doesn't have a positive effort value"

Reply:
fixed.